packages/cysync-core/src/dialogs/AddAccountGuide/Dialogs/Device/AddAccountSingleChainDialog.tsx

```diff
@@ -0,0 +1,247 @@
+import {
+  LangDisplay,
+  ScrollableContainer,
+  FlexGapContainer,
+  DialogBox,
+  DialogBoxHeader,
+  DialogBoxBody,
+  LeanBoxContainer,
+  LeanBox,
+  Typography,
+  Image,
+  InputLabel,
+  DialogBoxFooter,
+  Button,
+  settingsIcon,
+  Flex,
+  bitcoinIcon,
+} from '@cypherock/cysync-ui';
+import { Toggle } from '@cypherock/cysync-ui/dist/esm/components/atoms/Toggle';
+import React, { FC, useState } from 'react';
+
+import { selectLanguage, useAppSelector } from '~/store';
+
+import { useAddAccountGuide } from '../../context';
+
+const dataArray = [
+  {
+    id: '10',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 1',
+    checkBox: true,
+    tag: 'TAPROOT',
+  },
+];
+const accountNotSynced = [
+  {
+    id: '1',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 2',
+    checkBox: true,
+    tag: 'TAPROOT',
+  },
+  {
+    id: '2',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 2',
+    checkBox: true,
+    tag: 'TAPROOT',
+  },
+  {
+    id: '3',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 2',
+    checkBox: true,
+    tag: 'SEGWIT',
+  },
+  {
+    id: '4',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 2',
+    checkBox: true,
+    tag: 'NATIVE SEGWIT',
+  },
+];
+const accountsInPortfolio = [
+  {
+    id: '11',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 1',
+    checkBox: true,
+    tag: 'SEGWIT',
+  },
+  {
+    id: '12',
+    leftImageSrc: bitcoinIcon,
+    text: 'Bitcoin 1',
+    checkBox: true,
+    tag: 'NATIVE SEGWIT',
+  },
+];
+
+export const AddAccountSingleChainDialog: FC = () => {
+  const [isChecked, setIsChecked] = useState(false);
+  const { onNext } = useAddAccountGuide();
+  const [checkedItems, setCheckedItems] = useState<string[]>([]);
+
+  const handleCheckBoxChange = (id: string) => {
+    if (checkedItems.includes(id)) {
+      setCheckedItems(prevItems => prevItems.filter(item => item !== id));
+    } else {
+      setCheckedItems(prevItems => [...prevItems, id]);
+    }
+  };
+
+  const handleToggleChange = (checked: boolean) => {
+    setIsChecked(checked);
+  };
+
+  const handleClick = () => {
```

Comment:
The function name should describe its function.

```suggestion
  const handleDeselectAll = () => {
```

Reply:
the source has been modified a bit and these changes are applied there.